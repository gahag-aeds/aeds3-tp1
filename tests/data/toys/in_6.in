7
0.0000 0.9972 0.20858 0.68895 0.22309 0.61446 0.76968 
0.0028 0.0000 0.42028 0.04526 0.17342 0.72835 0.81755 
0.79142 0.57972 0.0000 0.65201 0.32367 0.8582 0.28183 
0.31105 0.95474 0.34799 0.0000 0.03679 0.32164 0.67647 
0.77691 0.82658 0.67633 0.96321 0.0000 0.93042 0.96423 
0.38554 0.27165 0.1418 0.67836 0.06958 0.0000 0.20618 
0.23032 0.18245 0.71817 0.32353 0.03577 0.79382 0.0000 
