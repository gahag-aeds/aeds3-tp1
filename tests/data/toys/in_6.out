0.114131 0.039068 0.160152 0.042285 0.584679 0.022708 0.036977 